src/vapid-helper.js

```diff
@@ -6,6 +6,12 @@ const asn1 = require('asn1.js');
 const jws = require('jws');
 const url = require('url');
 
+/**
+ * DEFAULT_EXPIRATION is set to seconds in 12 hours
+ */
+const DEFAULT_EXPIRATION_SECONDS = 12 * 60 * 60;
```

Comment:
We orginally had 24 hours but we hit an issue where some peoples servers would get into a bad state with web push services (i.e. I think one provided - probably FCM - would reject expirations over 24 hours in the future)